### PR TITLE
LPD-38600 Add z-index for context menu when maximized

### PR DESCRIPTION
**Pull request by fortunatomaldonado**
https://liferay.atlassian.net/browse/LPD-38600
https://liferay.atlassian.net/browse/LPP-55736

The context menu does not appear when CKEditor is maximized.
I added z-index to make the context menu appear.

Please let me know if there are any questions or comments about this.
Thank you!

### CODE REVIEW

**Comment by fortunatomaldonado** [comment]
ci:test:sf

**Comment by fortunatomaldonado** [comment]
ci:test:relevant

File: modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/main.scss [hunk 1]
```diff
@@ -263,6 +263,10 @@
 	z-index: 100 !important;
 }
 
+.cke_panel.cke_menu_panel {
+	z-index: 10001 !important;
+}
+
```
Comment (markocikos) [inline modules/apps/frontend-editor/frontend-editor-ckeditor-web/src/main/resources/META-INF/resources/css/main.scss:269]: All these `!important`s make me nervous. It is a bad pattern to start having them everywhere, as future overrides are now much harder. It is a joker of sorts to be used when we're out of all other options. The only saving grace here is it is just `z-index`, that already has messed up priorities handling.

**Review by markocikos — APPROVED** [review]
@fortunatomaldonado We can push this as it is, but I'm not too happy about overall way we're handling style priorities (more inline).

**Comment by fortunatomaldonado** [comment]
ci:forward

**Pull request closed without merging**